Source/WebCore/css/svg.css

```diff
@@ -84,3 +87,7 @@ text, tspan, tref {
     transform-origin: 0 0;
     transform-box: view-box;
 }
+
+:link, :visited {
+    cursor: pointer;
+}
```

Comment:
In `html.css` we have this (which looks weird, to be fair):

```
a:any-link {
    color: -webkit-link;
    text-decoration: underline;
    cursor: auto;
}

a:any-link:active {
    color: -webkit-activelink;
}

area:any-link {
    cursor: pointer;
}
```
I would expect these to be aligned in some way.

I think we want to at least use `:any-link` and tell the SVG WG to adopt that as well.

Reply:
Yes - I also checked both UA stylesheet for Firefox and Chrome and they have:

```
a:any-link {
    cursor: pointer; <- Only chrome has this; Firefox has `text-decoration: none`.
}

```
Should I change to above?

NOTE - our `html.css` should have also `cursor: pointer`. We fail WPT due to it, also @vitorroriz tried it in past but it started causing cursor issue on Amazon Prime.

Reply:
Plus if I look into HTML Spec as well, we have:

```:link, :visited { text-decoration: underline; cursor: pointer; }```

https://html.spec.whatwg.org/multipage/rendering.html#phrasing-content-3

Reply:
@annevk - https://github.com/whatwg/html/issues/10133

Reply:
Thanks, if we make this `:any-link` and remove the additional namespace declaration I think I'm okay with trying this out. I'd be interested in learning about the Amazon issue. Maybe something we need to raise with whatwg/html as well.

Reply:
> Thanks, if we make this `:any-link` and remove the additional namespace declaration I think I'm okay with trying this out. I'd be interested in learning about the Amazon issue. Maybe something we need to raise with whatwg/html as well.

@annevk - I made the changes and the @vitorroriz bug is here - https://bugs.webkit.org/show_bug.cgi?id=173909 and this comment from Brent (might be Radar link as well below it) would give more context - https://bugs.webkit.org/show_bug.cgi?id=173909#c6